apps/prairielearn/src/pages/instructorCourseAdminSettings/instructorCourseAdminSettings.html.ts

```diff
@@ -115,30 +150,44 @@ export function InstructorCourseAdminSettings({
                     The Github repository that can be used to sync course files.
                   </small>
                 </div>
-                <p class="mb-0">
-                  ${coursePathExists && courseInfoExists
-                    ? resLocals.authz_data.has_course_permission_view
-                      ? resLocals.authz_data.has_course_permission_edit &&
-                        !resLocals.course.example_course
-                        ? html`
-                            <a
-                              data-testid="edit-course-configuration-link"
-                              href="${resLocals.urlPrefix}/${resLocals.navPage}/file_edit/infoCourse.json"
-                            >
-                              Edit course configuration
-                            </a>
-                            in <code>infoCourse.json</code>
-                          `
-                        : html`
-                            <a
-                              href="${resLocals.urlPrefix}/${resLocals.navPage}/file_view/infoCourse.json"
-                            >
-                              View course configuration
-                            </a>
-                            in <code>infoCourse.json</code>
-                          `
+                  ${
+                    coursePathExists && courseInfoExists
+                      ? resLocals.authz_data.has_course_permission_view
+                        ? resLocals.authz_data.has_course_permission_edit &&
+                          !resLocals.course.example_course
+                          ? html`
+                              <button
+                                id="save-button"
+                                type="submit"
+                                class="btn btn-primary mb-2"
+                                name="__action"
+                                value="update_configuration"
+                              >
```

Comment:
Is there precedent for enabling the button only after changes have been made in the form? There was mention of this in the project description but I am not sure how to accomplish this. I can start writing a new script for it but thought I would check first.

Reply:
Follow up: The only similar element I found was in the `fileEditor` which did not behave exactly as we wanted here. I wrote a script in `/assets/scripts/saveButtonEnabling.ts` that checks the input fields for differences and then enables the save button if any fields have changed.